### DISCUSSION TITLE
Add Mode  Median and Export Functions

### DISCUSSION THREAD

**Discussion by probles40** [dcomtois/summarytools#158] [Q&A] [answered]
Support,

Would it be possible to include on your next release Mode, Median and Export results to look like SPSS?

Thanks

P.Robles

**Comment by dcomtois** [comment]
Hi, what summarytools function(s) are you referring to? And could you give an example of an SPSS export with notes about what specifically make them appealing to you?

**Comment by dcomtois** [comment, marked as answer]
There is a built in mechanism to customize the output for dfSummary. The document [Custom Statistics in dfSummary](https://github.com/dcomtois/summarytools/blob/master/doc/Custom-Statistics-in-dfSummary.pdf) shows how to do it. The help section for `st_options()` also explains how to set the `dfSummary.custom.1` and `dfSummary.custom.2` options.
